CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Change Log
 
+## 1.7.2
+
+- Comments syntax fix: use "//" instead of "#"
+- Proper check for qraphql document for uploadGraphqlSchema command
+- Added graphql domain setting
```

Comment:
```suggestion
- Revised comment syntax to use `//` instead of `#`.
- Improved GraphQL file type check for the `uploadGraphqlSchema` command.
- Added GraphQL domain configuration item.
```

Reply:
thanks for the suggestion!